docs/backup_workstations.rst

```diff
@@ -134,14 +130,23 @@ for each device you backup.
 Finally, once you have completed the steps described in this section for each
 USB drive, unmount the Backup partition and store the drive somewhere safely.
 
+Creating Backup Tails Drives
+----------------------------
+
+There are situations when you may need to create a new Tails Drive with your
+Persistent volumes saved. For example, you may find a USB is failing and would
+like to save your data. This is a separate workflow and the official
```

Comment:
Hmm, this sounds confusing as the purpose of the above workflow is to backup and save the data on the Tails drives. Perhaps instead of this paragraph, we can say in a note at the top of this document "This workflow will create a single USB drive with the backup data from all Tails drives. If instead you'd like to create a single duplicate backup Tails drive, you should follow the official documentation at... "

Reply:
@redshiftzero I am okay with a `.. note: ` at the top of this. I will explore tomorrow.